specification/security/_types/Privileges.ts

```diff
@@ -151,6 +151,10 @@ export enum ClusterPrivilege {
    * @availability stack
    */
   monitor_snapshot,
+  /**
+   * @availability stack
```

Comment:
Should we also add `since=8.17.0` here?

Reply:
done in https://github.com/elastic/elasticsearch-specification/pull/3125/commits/720e1f0b7cd3f408d799a4d698740056ca37b2f0